docs/contracts/using-for.rst

```diff
@@ -82,12 +100,13 @@ Let us rewrite the set example from the
         }
     }
 
-It is also possible to extend elementary types in that way:
+It is also possible to extend elementary types in that way.
```

Comment:
Strictly speaking, the array below is not an "elementary type", though, is it?
Not sure we need to change anything, though.